arduino-ide-extension/tslint.json

```diff
@@ -15,6 +15,7 @@
       "check-else",
       "check-whitespace"
     ],
+    "quotemark": [true, "single", "avoid-escape"],
```

Comment:
We used to use this rule in Theia:
```json
    "quotemark": {
      "options": [
        true,
        "single",
        "jsx-single",
        "avoid-escape",
        "avoid-template"
      ]
    },
```
👆 This will be obsolete once we switch to `@typescript-eslint/eslint-plugin`.